python/action_server/config.py

```diff
@@ -0,0 +1,66 @@
+from pydantic import BaseModel
+import zenoh
+import json
+from typing import Tuple
+
+zenoh.init_logger()
+
+class ZenohValidator(BaseModel):
+    '''
+    Config module for zenoh configuration. Validates the configuration variables using ConfZ by pydantic module.
+    Args:
+        mode(str): two choices were there:
+            - peer
+            - client
+        connect(str): the endpoint to connect
+        listen(str): the endpoint to listen 
+        config(str): a configuration file for zenoh variables.
+        base_key_expr(str): a common keyexpression to use. 
+    '''
+    mode: str
```

Comment:
```py
mode: str = field(default="peer")
```

Reply:
Use the method above to provide fallback values

Reply:
Switched to this method.